core/src/main/java/org/apache/calcite/tools/RelBuilder.java

```diff
@@ -2602,6 +2611,32 @@ private RelBuilder aggregate_(GroupKeyImpl groupKey,
     return project(projects.transform((i, name) -> aliasMaybe(field(i), name)));
   }
 
+  public static Set<Integer> getAllFields2(ImmutableBitSet groupSet,
+      List<AggregateCall> aggCallList, boolean checkSubquery, int expectedSelectListCount) {
```

Comment:
change this parameter name `checkSubquery` to `subqueryPresent`

Reply:
resolved